plugins/source/aws/resources/services/elastictranscoder/presets_fetch.go

```diff
@@ -0,0 +1,26 @@
+package elastictranscoder
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/elastictranscoder"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func fetchElastictranscoderPresets(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	c := meta.(*client.Client)
+	svc := c.Services().Elastictranscoder
+
+	p := elastictranscoder.NewListPresetsPaginator(svc, nil)
```

Comment:
I wonder why the AWS SDK supports a `StopOnDuplicateToken` option: should we perhaps be setting that to `true` to avoid infinite loops?

Reply:
Well, the option itself isn't implemented as a custom option. It might be an overkill to do it manually for all pagers, and the `List` resolvers don't do such logic.